src/test/java/cz/codecamp/logger/loggers/FileRotationLoggerTest.java

```diff
@@ -0,0 +1,42 @@
+package cz.codecamp.logger.loggers;
+
+import cz.codecamp.logger.LogLevelEnum;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.time.LocalDateTime;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by Lenovo on 18.10.2016.
+ */
+public class FileRotationLoggerTest {
+    FileRotationLogger frl;
+    @Before
+    public void setUp() {
+        frl = new FileRotationLogger(1,1,1);
+    }
+
+    @Test
+    public void setFileName() throws Exception {
+        LocalDateTime date = LocalDateTime.now();
+        assertEquals(frl.setFileName(date), String.valueOf(date.getDayOfMonth())+String.valueOf(date.getMonthValue())+String.valueOf(date.getYear())+String.valueOf(date.getHour())+String.valueOf(date.getMinute())+".log");
```

Comment:
Pokud jsou nějaké věci závislé na čase, nejlepší je čas nějak "vymockovat", např. udělat `interface DateProvider { Date get(); }` a v testech podstrčit implementaci tohodle interface, která vrací jeden určitý čas. Jak je to udělané tady, máš v podstatě zkopírovaný obsah té samotné metody do testu - je to pak taková tautologie, v podstatě testuješ, že chybu, kterou jsou udělal v jednom jsi udělal i v testu.